backend/sources/facebook/sender/src/main/java/co/airy/core/sources/facebook/Sender.java

```diff
@@ -88,18 +88,20 @@ private Message sendMessage(SendMessageRequest sendMessageRequest) {
 
             api.sendMessage(pageToken, fbSendMessagePayload);
 
+            //TODO move the change state logic to lib/kafka/schema once we have more sources
```

Comment:
Not to an avro message wrapper library in `backend/avro/message`? 🤔

Reply:
better place! I will update the comment